src/commands/e2e/WdioConfigTemplate.ts

```diff
@@ -143,6 +143,7 @@ exports.config = {
                 maxInstances: 1,
                 browserName: '${this.browser}',
                 'goog:chromeOptions': {
+                    w3c: false,
```

Comment:
do you only need w3c in headless ?
probably we need to introduce w3c option for chrome in normal mode as well?
I will add it hope that`s fine

Reply:
w3c is needed in any case